app/components/ChainTokenSelector.tsx

```diff
@@ -192,23 +173,29 @@ export const ChainTokenSelector = () => {
             onSelectionChange={handleNetworkSelectionChange}
             selectedKey={
               isOnrampActive
-                ? selectedPurchaseCurrencyNetwork?.display_name
+                ? selectedPurchaseCurrencyNetwork?.displayName
                 : selectedSellCurrencyNetwork?.display_name
             }
           >
-            {(
-              (isOnrampActive
-                ? selectedPurchaseCurrency?.networks
-                : selectedSellCurrency?.networks) || []
-            ).map((network) => (
-              <AutocompleteItem
-                key={network.display_name}
-                value={network.display_name}
-                className="capitalize"
-              >
-                {network.display_name}
-              </AutocompleteItem>
-            ))}
+            {isOnrampActive
+              ? (selectedPurchaseCurrency?.networks || []).map((network) => (
```

Comment:
nit: I think it'd be easier to read if this were something like the following @rustam-cb 

```ts
const networkDisplayNames = useMemo(() => {
  if  (isOnrampActive) {
    return (selectedPurchaseCurrency.networks ?? []).map(({ displayName }) => displayName);
  }
  
  return (selectedSellCurrency.networks ?? []).map(({display_name}) => display_name);
}, [])

// ... 

{networkDisplayNames.map((networkDisplayName) => {
 // ...
})}
```